pokemongo_bot/cell_workers/seen_fort_worker.py

```diff
@@ -97,10 +108,9 @@ def work(self):
                         logger.log('- ' + str(item_count) + "x " + item_name + " (Total: " + str(self.bot.item_inventory_count(item_id)) + ")", 'yellow')
                 else:
                     logger.log("[#] Nothing found.", 'yellow')
-
-                pokestop_cooldown = spin_details.get(
-                    'cooldown_complete_timestamp_ms')
```

Comment:
You're ignoring the cooldown that the game is giving you, why? Shouldn't you get this cooldown and add a random delta to it?